components/review-sections/IdAndResidencyReviewSection.tsx

```diff
@@ -0,0 +1,83 @@
+import { Heading, HeadingLevels, SummaryList } from "lbh-frontend-react";
+import React from "react";
+import PropTypes from "../../helpers/PropTypes";
+import useReviewSectionRows from "../../helpers/useReviewSectionRows";
+import {
+  idAndResidencyProcessSteps,
+  idAndResidencyResidentSteps,
+} from "../../steps/id-and-residency";
+import Storage from "../../storage/Storage";
+
+interface Tenant {
+  fullName: string;
+  id: string;
+  isPresent: boolean;
+}
+interface IdAndResidencyReviewSectionProps {
+  tenants: Tenant[];
+}
+
+interface TenantSectionProps {
+  tenant: Tenant;
+}
+
+const TenantSection: React.FunctionComponent<TenantSectionProps> = (props) => {
+  const { tenant } = props;
+  const { id, fullName, isPresent } = tenant;
+  return (
+    <>
+      <Heading level={HeadingLevels.H3}>
+        {fullName}
+        {isPresent ? "" : " (not present)"}
+      </Heading>
+      <SummaryList
+        rows={useReviewSectionRows(
+          Storage.ResidentContext,
+          idAndResidencyResidentSteps,
+          id
+        )}
+      />
+    </>
+  );
+};
+
+export const IdAndResidencyReviewSection: React.FunctionComponent<IdAndResidencyReviewSectionProps> = (
+  props
+) => {
+  const { tenants } = props;
+
+  return (
+    <>
+      <Heading level={HeadingLevels.H2}>
+        {"ID, residency, and tenant information"}
+      </Heading>
+      <SummaryList
+        rows={useReviewSectionRows(
+          Storage.ProcessContext,
+          idAndResidencyProcessSteps
+        )}
+      />
```

Comment:
What does this render if the hook returns an empty array?

Reply:
This is the only thing that's left that's a little off - `presentForCheck` is the only `idAndResidencyProcessStep`, and is always coming back as an empty array for me. In the sub-component I'm rendering "Loading..." now if the ID steps come back as empty, but for some reason this is always empty...